README

```diff
@@ -34,6 +34,56 @@ xtensa-byt-elf-objcopy -I binary -O elf32-xtensa-le -B xtensa image.bin image.bi
 
 xtensa-byt-elf-objdump -D image.bin.elf > image.dis.txt
 
+
+logger
+======
+
+logger is used to print logs delivered from FW dma_trace mechanism, by searching log
+entries in *.ldc file generated by rimage
```

Comment:
Before giving the examples, would you please give a list of all parameters? like this 

Usage: sof_logger [OPTIONS]...
Options:
-h              help
-l,              specify the ldc file
-t               get traces from "/sys/kernel/debug/sof/trace", instead of the default "/sys/kernel/debug/sof/etrace"
-o              specify the output file, instead of the stdout
-i                ...

Examples:
- Get traces from "/sys/kernel/debug/sof/etrace" file and print logs to stdout
   $ sof_logger -l ldc_file

- Get gets traces from "/sys/kernel/debug/sof/etrace" file and prints logs to a specified output file
   $logger -l ldc_file -o out_file
...

Reply:
@mengdonglin I've updated that.